pkg/api/universal/scheduler.go

```diff
@@ -29,7 +29,7 @@ func (a *Universal) ScheduleJob(ctx context.Context, inReq *runtimev1pb.Schedule
 	// validate job details, date, schedule, etc??
 	var err error
 
-	metadata := map[string]string{"app_id": a.AppID()}
+	metadata := map[string]string{"appId": a.AppID()}
```

Comment:
A quick grep in the project makes me believe `app_id` is what we want here

Reply:
Where is this being consumed?

Reply:
I found the references and they are for metrics - which follows its own convention. We need to decide the convention we want to use for our metadata in Dapr. I did not find anything specifying which casing convention we want to use.